src/services/ros/index.ts

```diff
@@ -29,10 +29,26 @@ export interface IUser {
   isAdmin: boolean;
   accounts: IAccount[];
   metadata: IUserMetadataRow[];
+  status: UserStatus;
 }
 
 export type User = IUser & Realm.Object;
 
+export enum UserStatus {
+  active = 'active',
+  blacklisted = 'blacklisted',
+}
+
+export function humanizeUserStatus(userStatus: UserStatus) {
+  switch (userStatus) {
+    case UserStatus.blacklisted:
+      return 'Blacklisted';
+    case UserStatus.active:
+    default:
+      return 'Active';
```

Comment:
I suggest defaulting to the raw `userStatus` as defaulting to "Active" could be really confusing.

Reply:
I did it like that because we don't have a migration that will set everyone's status to `Active` and wanted to handle the empty/null case. But I get your point - in the future, if we introduce a new status on the server, people with old studio versions may see confusing info. So I'll explicitly handle the `null` and empty string values as `Active` and default to the raw status for others.

Reply:
We don't have a migration for statuses so by default everything will be null.